Exercise_2/main.cpp

```diff
@@ -0,0 +1,179 @@
+#include <string>
+#include <locale>
+#include <iostream>
+#include <fstream>
+#include <algorithm>
+#include <vector>
+#include <limits>
+#include <climits>
+#include <list>
+
+using namespace std;
```

Comment:
It's not recommended to do this. The `std` namespace has a lot of names that can conflict with your names, e.g. `min`, `max`, `abs` and a lot more.
It's safer to just not use `using namespace ...` and type the full name like `std::vector<>`